testutils/evm.go

```diff
@@ -5,11 +5,30 @@
 package testutils
 
 import (
+	"context"
+	"crypto/ecdsa"
 	"encoding/hex"
 	"errors"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
```

Comment:
Our original plan was not to import the `github.com/ethereum/go-ethereum`  library

Reply:
we will refactor the repos structure to resolve it.